tests/src/allowance.rs

```diff
@@ -53,64 +54,80 @@ fn should_approve_funds_contract_to_contract() {
 fn should_approve_funds_account_to_account() {
     let (mut builder, test_context) = setup();
 
+    let default_account_user_key = Key::Account(*DEFAULT_ACCOUNT_ADDR);
+    let (_, account_user_1_account_hash, _) = get_test_account("ACCOUNT_USER_1");
+    let account_user_1_key = Key::Account(account_user_1_account_hash);
+
     test_approve_for(
         &mut builder,
         &test_context,
-        Key::Account(*DEFAULT_ACCOUNT_ADDR),
-        Key::Account(*DEFAULT_ACCOUNT_ADDR),
-        Key::Account(*ACCOUNT_1_ADDR),
+        default_account_user_key,
```

Comment:
seems the same as as *DEFAULT_ACCOUNT_KEY ?

Reply:
Can you maybe replace every occurrences in every files  ?
```
Key::Account(*DEFAULT_ACCOUNT_ADDR)
```
by 

```
*DEFAULT_ACCOUNT_KEY
```

Reply:
Nope, currently [DEFAULT_ACCOUNT_KEY is the exact same value as DEFAULT_ACCOUNT_ADDR](https://github.com/casper-network/casper-node/blob/dev/execution_engine_testing/test_support/src/lib.rs#L97), so it's not a key but an AccountHash